### PR TITLE
Loading medibase and icd11 static json and migrations

### LINKED ISSUE #1817: Load medibase and icd11 from static json
medibase and icd 11 are static data which could be loaded to directly to cache without fetching them from the db

- update the medibase model to use atc_classification key as unique id
- migrate all related models to use this id as fk
- make id(atc_classification) as pk 
- remove external_id
- update the load redis index functions to directly use the json to populate the index

- ~make loading static data run parallel to the migration in the beat container~

cc: @rithviknishad

### LINKED ISSUE #2003: Clear entries from the search index if they are not in db anymore
**Describe the bug**

we are having multiple dolo in the medicine of the patient consultation

**To Reproduce**

**Expected behavior**

**Screenshots**

image

### PR DESCRIPTION
**Pull request by DraKen0009**
## Proposed Changes

1.Updated medibase.json
- removed _id field
- added id field

2.Updated `MedibaseMedicine` model , removed external_id

3.Updated  `python manage.py load_redis_index` command to directly use the json to populate the redis

4.Migrate all related models to use id as fk instead of external_id 

### Associated Issue

- Fixes #1817 
- Fixes #2003

## Merge Checklist

- [x] Tests added/fixed
- [x] Linting Complete

_*Only PR's with test cases included and passing lint and test pipelines will be reviewed*_

@coronasafe/care-backend-maintainers @coronasafe/care-backend-admins

### CODE REVIEW

File: care/facility/management/commands/load_medicines_data.py [hunk 1]
```diff
@@ -34,6 +35,19 @@ def handle(self, *args, **options):
...
             ],
-            batch_size=1000,
-            ignore_conflicts=True,
+            batch_size=5000,
+            update_conflicts=True,
+            unique_fields=["name"],
+            update_fields=[
+                "created_date",
+                "modified_date",
+                "deleted",
```
Comment (sainak) [inline care/facility/management/commands/load_medicines_data.py]:
```suggestion
```

File: care/facility/management/commands/load_medicines_data.py [hunk 2]
```diff
@@ -24,6 +24,7 @@ def handle(self, *args, **options):
         MedibaseMedicine.objects.bulk_create(
```
Comment (sainak) [inline care/facility/management/commands/load_medicines_data.py]: add a check to fallback to bulk create only if there are no medicines in the db to speed up the process

File: care/facility/migrations/0416_set_static_id_to_medicines.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    Prescription = apps.get_model("facility", "Prescription")
+
+    if MedibaseMedicine.objects.count() == 0:
+        return
+
+    with open("data/medibase.json", "r") as f:
+        data = json.load(f)
+
+    for medicine in data:
+        m = MedibaseMedicine.objects.get(name=medicine["name"])
```
Comment (sainak) [inline care/facility/migrations/0416_set_static_id_to_medicines.py]: create in case of DoesNotExists error

**Comment by vigneshhari** [comment]
@sainak either complete or close this MR.

**Pull request closed without merging**